hbp_knowledge/anatabine/paris2013.bel

```diff
@@ -146,115 +160,149 @@ anatabine (Mann–Whitney U=0, Z=-2.309, P=0.021) and a significant
 inhibition of p65 NFkB phosphorylation was observed with
 600 mg/ml of anatabine (Mann–Whitney U=1.0, Z=-2.021,
 P=0.043) and with 800 mg/ml of anatabine (Mann–Whitney U=0,Z=-2.309, P=0.021)"
+SET Confidence = "High"
 SET CellLine = "SH-SY5Y"
 # Dosage: 600 mg/ml and 800 mg/ml anatabine
 a(CHEBI:Anatabine) -| p(HGNC:STAT3,pmod(Ph))
 a(CHEBI:Anatabine) -| p(HGNC:RELA,pmod(Ph))
 UNSET CellLine
+UNSET Confidence
 
 SET Evidence = "We next tested the impact of anatabine on STAT3 and p65
 NFkB phosphorylation induced by a 24 h treatment with LPS on
 human microglial cells, a cell type known to express alpha7-nicotinic
 acetylcholine receptor subtype (Suzuki et al., 2006)"
+SET Confidence = "Medium"
+SET Species = "9606"
 SET Cell = "microglial cell"
 a(MESH:Microglia) -> a(MESH:"alpha7 Nicotinic Acetylcholine Receptor")
 UNSET Cell
+UNSET Species
+UNSET Confidence
 
+#not sure of information provide: was LPS given with anatabine?
 SET Evidence = "An increased
 STAT3 (Mann–Whitney U=0, Z=-2.309, P=0.021) and p65 NFkB
 phosphorylation (Mann–Whitney U=0, Z=-2.309, p=0.021) was
 observed in LPS treated microglia (Fig. 3)"
+#SET Confidence
 SET Cell = "microglial cell"
 a(CHEBI:lipopolysaccharide) -> p(HGNC:STAT3,pmod(Ph))
 a(CHEBI:lipopolysaccharide) -> p(HGNC:RELA,pmod(Ph))
 UNSET Cell
+#UNSET Confidence
 
 SET Evidence = "We observed that anatabine significantly suppressed the stimulation of p65 NFkB and
 STAT3 phosphorylation by LPS in microglia (Fig. 3)"
+#SET Confidence = "High"
```

Comment:
To encode the exact process mentioned here, a nested statement has to be used
a(CHEBI:Anatabine) -| (a(CHEBI:lipopolysaccharide) -> p(HGNC:STAT3,pmod(Ph)))
something like this...
because we are not supposed to use nested statements I only coded the final results ( increase of phoshorylation with LPS, and decrease when anatabine is added)
This is not 100% correct but the best way to encode this